suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -438,3 +438,10 @@ export const networksCompatibility: Network[] = Object.entries(networks).flatMap
         })),
     ],
 );
+
+export const getMainnets = () => networksCompatibility.filter(n => !n.accountType && !n.testnet);
+
+export const getTestnets = (debug = false) =>
+    networksCompatibility.filter(
+        n => !n.accountType && n.testnet === true && (n.symbol !== 'regtest' || debug),
```

Comment:
are we sure we want to show testnet only in debug mode? I think QA use it quite often

Reply:
I did not make any modifications to the function logic but only moved the logic to functions. If you look at the file https://github.com/trezor/trezor-suite/pull/7165/files/0ecb2a78d8ca37ded961d59fed0f3d935a765867#diff-0908abe1e0b2ced62667f9b9532f4ab1873c50978e685a18900f73a2a30b0a63 you will see there is the same condition like now in diff.